File: 백준/Silver/1475. 방 번호/방 번호.js
const input = require('fs').readFileSync(process.platform == 'win32' ? "./input.txt" : '/dev/stdin').toString().trim().split("").map(Number);

const solution = () => {
    const arr = Array(10).fill(0);

    for (num of input) {
        if (num == 6 || num == 9) {
            if (arr[6] > arr[9]) { arr[9]++; }
            else { arr[6]++; }
        } else {
            arr[num]++;
        }
    }
    console.log(Math.max(...arr));
    
}

solution()

File: 백준/Silver/1475. 방 번호/방 번호.py
n = input()
arr = [0] * 10

for i in range(len(n)):
  num = int(n[i])
  if num == 6 or num == 9:
    if arr[6] <= arr[9]:
      arr[6] += 1
    else:
      arr[9] += 1
  else:
    arr[num] += 1
print(max(arr))